live-examples/js-examples/arraybuffer-slice.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(16);
+
+var int32View = new Int32Array(buffer); // create the view
+int32View[2] = 42;
+
+var lastHalfView = new Int32Array(buffer.slice(8,16));
+
+console.log(lastHalfView[0]);
+
+// expected output: 42
```

Comment:
Nit: Some examples have an empty line between the `console` statement and the expected output line, and some do not. For consistency, and taking up less vertical space, lets go with, no empty line. Thoughts?

Reply:
Sure. That's fine. Will rectify now :)